package.json

```diff
@@ -47,13 +47,15 @@
     ]
   },
   "devDependencies": {
+    "cypress": "^8.4.1",
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-json": "^3.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.25.1",
-    "prettier": "2.4.0"
+    "prettier": "2.4.0",
+    "typescript": "^4.4.3"
```

Comment:
I really like CucumberJS in addition to Cypress. Is this something you can add? If not it can be added in another PR

Reply:
Cucumber is not something I am comfortable with quite yet and would take a few days IU think to implement by me. Maybe something I could do for Hacktober or something.